cpp/src/neighbors/detail/vamana/vamana_build.cuh

```diff
@@ -388,6 +388,7 @@ auto quantize_all_vectors(raft::resources const& res,
 {
   auto dim         = residuals.extent(1);
   auto vq_codebook = raft::make_device_matrix<float, uint32_t, raft::row_major>(res, 1, dim);
+  raft::linalg::map_offset(res, vq_codebook.view(), [] __device__(size_t i) { return 0; });
```

Comment:
Can we use `raft::matrix::fill` for this?

Reply:
Sure, I'll update this and verify correctness.